ConsistencyManager/ConsistencyManager.swift

```diff
@@ -140,23 +140,23 @@ open class ConsistencyManager {
      Note that calling this method on a paused listener will not unpause it.
      - parameter listener: The consistency manager listener that is listening to a model
     */
-    open func listenForUpdates(_ listener: ConsistencyManagerListener) {
+    open func addListener(_ listener: ConsistencyManagerListener) {
```

Comment:
Yeah. This was a big question, and found two counter examples:

One is from the swift API style guidelines where they recommend: `addObserver(_ observer...`
The other is from the initial rules saying you should use a named first parameter when:
```
methods and functions where the first parameter of a method is defaulted
methods and functions where the first argument uses a prepositional phrase
methods and functions that implement factory methods
methods and functions where method arguments represent a split form of a single abstraction
```
Since this doesn't meet any of the criteria, I decided on `addListener(_ listener)`